SimpleWAWS/Code/CsmExtensions/CsmSiteExtensions.cs

```diff
@@ -128,13 +128,17 @@ public static async Task<Site> Load(this Site site, CsmWrapper<CsmSite> csmSite
 
             if (!site.IsSimpleWAWSOriginalSite)
             {
-                site.AppSettings["FUNCTIONS_EXTENSION_VERSION"] = SimpleSettings.FunctionsExtensionVersion;
+                site.AppSettings["FUNCTIONS_EXTENSION_VERSION"] = Constants.FunctionsVersion;
```

Comment:
Agree on using SimpleSettings instead of hardcoded constant for functions version and Node version. We should update the FunctionsExtensionVersion appsetting to ~2 after deployment.

 I think we should update FunctionsContainerDeployedVersion to new value-this is not coming from appsetting, so that after deployment newly created RG has new value.